configure.ac

```diff
@@ -730,7 +730,7 @@ AC_SUBST(CONFIG_ECMASCRIPT_SMJS_HEARTBEAT)
 # ===================================================================
 
 AC_ARG_ENABLE(sm-scripting,
-	      [  --disable-sm-scripting  ECMAScript browser scripting (requires Spidermonkey)],
+	      [  --disable-sm-scripting  disable SpiderMonkey Mozilla Javascript engine],
 	      [if test "$enableval" != no; then enableval="yes"; fi
```

Comment:
sm-scripting is similar to lua, perl, python or ruby scripting, so this message could be better.